src/routes/e2e/widget/inline/+page.svelte

```diff
@@ -18,10 +18,10 @@
     userResponse = null;
   }
 
-  form.onMount((component) => console.log(component));
+  form.onMount((component: HTMLDivElement) => console.log(component));
   // TODO: Use a more specific type
-  journey.onSuccess((response) => (userResponse = response?.user));
-  journey.onFailure((response) => console.log(response.journey?.error));
+  journey.onSuccess((response: Response) => (userResponse = response?.user));
+  journey.onFailure((response: Response) => console.log(response.journey?.error));
```

Comment:
Anytime we have to type a callback parameter like this, I would consider it a bug, just FYI.

Reply:
Understood.